### PR TITLE
Fix PHP Warning:  strpos(): Empty needle

### PR DESCRIPTION
**Pull request by strk**
when no notification is pending...

### CODE REVIEW

File: mod/ping.php [hunk 1]
```diff
@@ -357,9 +357,11 @@ function ping_get_notifications($uid) {
 			// Replace the name with {0} but ensure to make that only once
 			// The {0} is used later and prints the name in bold.
 
-			$pos = strpos($notification["message"],$notification['name']);
-			if ($pos !== false)
-				$notification["message"] = substr_replace($notification["message"],"{0}",$pos,strlen($notification["name"]));
+			if ( $notification['message'] ) {
```
Comment (annando) [inline mod/ping.php]: Could you remove the spaces inside the brackets so that it looks this way:

```
if ($notification['message']) {
```

Thanks!

Comment (strk) [inline mod/ping.php]: Could you please tweak it yourself ?
I understand pushing the button is convenient, but then maybe
add a pre-commit hook applying the style of preference ?

**Comment by strk** [comment]
Actually, forget about this. I was almost 2K commits behind when I wrote it. It could very well be there's no WARNING to fix.

I'll open a new one if needed

**Comment by annando** [comment]
And I just realized that this pull request went against the master branch. When you do pull requests, please be on the "develop" branch and do the pull requests against the master branch as well.

**Pull request closed without merging**